packages/babel-helper-compilation-targets/src/index.js

```diff
@@ -207,9 +211,10 @@ export default function getTargets(
     }
 
     const browsers = browserslist(browsersquery, {
-      path: options.configPath,
+      config: options.config,
+      path: options.path ?? options.configPath,
       mobileToDesktop: true,
-      env: options.browserslistEnv,
+      env: options.env ?? options.browserslistEnv,
```

Comment:
Did you mean
```suggestion
      env: options.browserslistEnv ?? options.env,
```
? Otherwise it is a breaking change.